test/log_helper.erl

```diff
@@ -0,0 +1,20 @@
+%% Helper module for checking expected log messages
+%% For assertions, see log_helper.hrl
+
+-module(log_helper).
+-compile([export_all, nowarn_export_all]).
+
+%% Call in init_per_testcase
+set_up() ->
+    logger:add_handler(handler_id(), ?MODULE, #{receiver => self()}).
+
+%% Call in end_per_testcase
+tear_down() ->
+    logger:remove_handler(handler_id()).
+
+%% Logger callback
+log(Event, #{receiver := Receiver}) ->
```

Comment:
https://github.com/esl/cets/blob/main/test/cets_SUITE.erl#L948 does the same.

Reply:
I wasn't aware of that tool. It looks quite complicated. I'd stick with the simple solution for now.